### PR TITLE
all policies create button & added emptystate views to empty resources

### LINKED ISSUE #107: All Policies: add a Create button
Add a "Create" button to the "All Policies" page, with a multi-select to pick a policy kind to create

### PR DESCRIPTION
**Pull request by jasonmadigan**
Resolves #107

Screenshot 2024-10-17 at 17 57 20

### CODE REVIEW

File: locales/en/plugin__kuadrant-console-plugin.json [hunk 1]
```diff
@@ -78,19 +77,15 @@
...
-  "Remove Match Expression": "Remove Match Expression",
-  "Remove Match Label": "Remove Match Label",
-  "Remove Value": "Remove Value",
-  "Routing Strategy": "Routing Strategy",
-  "Routing Strategy to use": "Routing Strategy to use",
   "Save": "Save",
   "Select a gateway": "Select a gateway",
-  "Select a Namespace": "Select a Namespace",
+  "Select a protocol": "Select a protocol",
+  "Select a Protocol": "Select a Protocol",
```
Comment (R-Lawton) [inline locales/en/plugin__kuadrant-console-plugin.json:88]: 2 feels like a bit overkill

Comment (jasonmadigan) [reply locales/en/plugin__kuadrant-console-plugin.json:88]: Got to make absolutely sure

(this was generated by the `yarn i18n` task, looks like an oddity. I'll replace one of them.)

Comment (jasonmadigan) [reply locales/en/plugin__kuadrant-console-plugin.json:88]: fixed

File: src/components/dnspolicy/LoadBalancingField.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ const LoadBalancingField: React.FC<LoadBalancingProps> = ({ loadBalancing, onCha
           }
           isRequired
           type="number"
-          placeholder={t("0")}
```
Comment (jasonmadigan) [inline src/components/dnspolicy/LoadBalancingField.tsx:34]: ó

Comment (R-Lawton) [reply src/components/dnspolicy/LoadBalancingField.tsx:34]: ah yes zero in non english languages qui qui

**Pull request merged**